nav2_smac_planner/lattice_primitives/trajectory.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021, Matthew Booker 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License. Reserved.
+
+from dataclasses import dataclass
+
+import numpy as np
+
+from helper import angle_difference
+
+
+@dataclass(frozen=True)
+class TrajectoryParameters:

```

Comment:
As a general comment across all files: have a block comment in any class explaining what the class does

Reply:
There are many without it